homeassistant/components/ted5000/sensor.py

```diff
@@ -55,6 +57,20 @@ def setup_platform(
     name = config.get(CONF_NAME)
     url = f"http://{host}:{port}/api/LiveData.xml"
 
+    # Register a repair issue that the integration is deprecated
+    async_create_issue(
+        hass,
+        DOMAIN,
+        "deprecated_ted5000_integration",
+        breaks_in_ha_version="2023.4.0",
```

Comment:
This was an arbitrary release. I'm not sure when is the proper time to schedule the removal of `ted5000`.

Reply:
As I remember it's minimum 3 releases after the code release.

So it's good now if this PR is merged for `2023.1.0`

Reply:
Perfect! I'll keep that in mind if the PR is merged after the January release.